napari_aicssegmentation/_custom_widgets.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+from qtpy.QtCore import Qt
+from qtpy.QtGui import QPixmap
+from qtpy.QtWidgets import (
+    QFormLayout,
+    QFrame,
+    QHBoxLayout,
+    QLabel, 
+    QWidget
+)
+
+DIR = os.path.dirname(__file__)
+
+
+""" 
+Create a warning message with a yellow warning sign icon on the left.
+
+Inputs:
+    message:    String
+Output:
+    A QWidget displaying a warning symbol and a message next to it
+"""
+
+
+def warning_message(message):
```

Comment:
I think it will be more consistent and testable to create custom widgets as classes instead of functions. This will be necessary for more complex widgets like the workflow step or the collapsible box that actually have some functionality.
I don't think you need to worry about fixing that right now though, can probably do it in the main codebase when we implement the page for real. 
The first PR we'll merge to `main` will likely have a `/widgets` folder with the warning message as our first custom widget.